integration_tests/pages/page.ts

```diff
@@ -814,4 +818,51 @@ export default abstract class Page {
 
     this.clearInput(name)
   }
+
+  shouldShowCalendarKey(type: 'twoColour' | 'threeColour'): void {
```

Comment:
Is this technically the same as the `verbose` argument in `shouldShowCalendar`? In fact, this could probably be integrated into `shouldShowCalendar`, as I don't think we ever show a calendar without a key?